config/src/config/test_data/public_full_node.yaml

```diff
@@ -1,18 +1,33 @@
 base:
+    # Update this value if you would like to connect to a different network
     chain_id: "TESTING"
+    # Update this value to the location you want Libra to store its database
     data_dir: "/opt/libra/data"
     role: "full_node"
     waypoint:
+        # Update this value to that which Libra publicly provides
         from_config: "0:01234567890ABCDEFFEDCA098765421001234567890ABCDEFFEDCA0987654210"
 
 execution:
-    genesis_file_location: "relative/path/to/genesis"
+    # Update this to the location to where the genesis.blob is stored, prefer fullpaths
```

Comment:
we should also mention if they want to catch up a waypoint without new genesis, set it to ""

Reply:
that's confusing... shouldn't I catch up regardless?

Reply:
how do you differentiate a case when the genesis is set and is supposed to be applied vs catch up?
imagine if the local version is N, waypoint is N+1, and the genesis is set but not update.

Reply:
I imagine some software engineering... can we not detect this in the format for genesis? That it is expected to be applied at Version N and therefore we should have a waypoint of version  > N? We should also be able to detect if a genesis has been applied in the past. If it has been applied, skip it, if it hasn't and the waypoint is bad, we should crash.

Reply:
> We should also be able to detect if a genesis has been applied in the past

This is quite expensive afaiu, also there's no guarantee such a genesis won't be applied again (in theory).

The format of the genesis doesn't tell whether we expect to apply it or not because it's just a transaction unless it's explicitly specified somewhere (either by empty path or specify in the waypoint config (apply or catch up)).